_includes/breadcrumbs.html

```diff
@@ -0,0 +1,36 @@
+<nav id="wb-bc" property="breadcrumb">
+  <h2>{{ site.data.i18n.general.head.youarehere[page.lang] }}:</h2>
+  <div class="container">
+    <ol class="breadcrumb">
+      <li><a href="./index.html">{{ site.data.i18n.general.head.home[page.lang] }}</a></li>
+
+      {%- if page.ref == 'docs' -%}
+        <li><span style="margin-left:10px">{{ site.data.i18n.general.nav.doc[page.lang] }}</span></li>
+      {%- endif -%}
+
+      {%- if page.ref == 'code' -%}
+        <li><span style="margin-left:10px">{{ site.data.i18n.general.nav.code[page.lang] }}</span></li>
+      {%- elsif page.ref == 'software' -%}
+        <li><span style="margin-left:10px">{{ site.data.i18n.general.nav.software[page.lang] }}</span></li>
+      {%- elsif page.ref == 'standard' -%}
+        <li><span style="margin-left:10px">{{ site.data.i18n.general.nav.standard[page.lang] }}</span></li>
+      {%- endif -%}
+
+      {%- if page.ref == 'codeForm' or page.ref == 'codeGroups' or page.ref == 'codeSchema' -%}
+        <li><a href="{{ site.data.i18n.general.nav.code.link[page.lang] }}">{{ site.data.i18n.general.nav.code[page.lang] }}</a></li>
+      {%- elsif page.ref == 'softwareForm' or page.ref == 'softwareSchema' -%}
+        <li><a href="{{ site.data.i18n.general.nav.software.link[page.lang] }}">{{ site.data.i18n.general.nav.software[page.lang] }}</a></li>
+      {%- elsif page.ref == 'standardForm' or page.ref == 'standardSchema' -%}
+        <li><a href="{{ site.data.i18n.general.nav.standard.link[page.lang] }}">{{ site.data.i18n.general.nav.standard[page.lang] }}</a></li>
+      {%- endif -%}
+
+      {%- if page.ref == 'codeForm' or page.ref == 'softwareForm' or page.ref == 'standardForm' -%}
+        <li><span style="margin-left:10px">{{ site.data.i18n.general.nav.form[page.lang] }}</span></li>
+      {%- elsif page.ref == 'codeGroups' -%}
+        <li><span style="margin-left:10px">{{ site.data.i18n.general.nav.groups[page.lang] }}</span></li>
+      {%- elsif page.ref =='codeSchema' or page.ref == 'softwareSchema' or page.ref == 'standardSchema' -%}
+        <li><span style="margin-left:10px">{{ site.data.i18n.general.nav.schema[page.lang] }}</span></li>
+      {%- endif -%}
```

Comment:
We don't need this last part of the breadcrumb since it just repeats the title of the page

Reply:
In that case, I suggest remove the breadcrumbs altogether. The site's navigation is not super complex and each page title is the same as the breadcrumb path, for instance, in "Form with validation for open source code", we got both "form" and "code". Even for top level pages (code, software, standard, doc), we already have the dark blue menu that can be used for navigating between them.

Reply:
Before removing the breadcrumb we should check with the re-design.

But following the canada.ca WET Style Guide, the breadcrumb doesn't include the current page

Reply:
However, if we still want to keep them, I don't think the last link are unnecessary. From my personal experience and all the example I just went through, breadcrumbs usually have a node showing where you are now, even if it repeat's the page title.

Reply:
I'll remove the last node for now, and we'll see with redesign what we do next.